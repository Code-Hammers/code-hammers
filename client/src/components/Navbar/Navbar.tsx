import React from "react";

const Navbar: React.FC = () => {
  return <div className="w-full max-w-xs">NAVBAR</div>;
};

export default Navbar;
