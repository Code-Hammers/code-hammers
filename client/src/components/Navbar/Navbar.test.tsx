import React from "react";
import TestRenderer from 'react-test-renderer'; 
import Navbar from "./Navbar";

describe('Navbar Component', () => {
    it('renders correctly', () => {
        const tree = TestRenderer
            .create(<Navbar />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});