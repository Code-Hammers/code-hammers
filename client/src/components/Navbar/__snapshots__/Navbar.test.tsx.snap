// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Component renders correctly 1`] = `
<div
  className="w-full max-w-xs"
>
  NAVBAR
</div>
`;
